async_simple/coro/Lazy.h

```diff
@@ -509,6 +509,10 @@ class [[nodiscard]] RescheduleLazy {
             launchCoro(std::move(*this), std::forward<F>(callback));
     }
 
+    void detach() {
+        start([](auto&&) {});
```

Comment:
in previous issue #17 we know user forget to handle exception with `start([](auto&&) {});`
we have to rethrow the existing exception in `detach` api, because user can not use Try type to handle it any more

```cpp
start([](auto&& t) {
  if (t.hasError()) {
    // rethrow
  }
});
```

Reply:
OK